feat: Add Team Growth back in (#13422)
PostHog/posthog.com: contents/teams/growth/index.mdx

---
title: Growth
sidebar: Handbook
showTitle: true
hideAnchor: false
template: team
---

## What the Growth team does

> We operate under a double mandate - albeit one easier than the FED's: increase top of funnel acquisition while increasing cross-product activation.

Reviewer: ```suggestion
We operate under a triple mandate - albeit one easier than the FED's: increase top of funnel acquisition, ensure successful activation, and increase cross-sell between products.
```

 
- Activation
  - Global onboarding flow
  - Develop tooling for individual product onboarding and activation
  - Help product teams leverage the tooling to create best-in-class onboarding & activation flows for their products

>   - Improve cross-product pollination

Reviewer: ```suggestion
```

- Product led growth
  - Develop new products and strategies to increase top of funnel

>   - Integrate with external partners to drive growth

Reviewer: ```suggestion
- Product led growth
  - Develop new products and strategies to increase top of funnel
  - Integrate with external partners to drive growth
- Product cross-sell
  - Track and measure cross-sell between products
  - Identify the right time to cross-sell different products to different users
  - Work with marketing to experiment on cross-sell initiatives outside the app
  - Build bridges to other products that appear when customers might benefit from them
```
